setup.py

```diff
@@ -45,7 +45,7 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket == 1.19.0rc0",
+        "pytket == 1.20.0rc1",
```

Comment:
1.19.1 is more recent if you want to use the latest.

Reply:
The barrier change is not in 1.19.1

Reply:
Oh I thought it was but you are right.